src/components/button/index.md.njk

```diff
@@ -50,6 +50,28 @@ Only use disabled buttons if research shows it makes the user interface easier t
 
 {{ example({group: "components", item: "button", example: "disabled", html: true, nunjucks: true, open: false}) }}
 
+### Stop users from accidentally sending information more than once
+
+Sometimes, users double click buttons because they:
+
+- have used operating systems where they have to double click items to make them work
+- are experiencing a slow connection which means they are not given feedback on their action quickly enough
+- have motor impairments such as hand tremors which cause them to click the button involuntarily 
+
+In some cases, this can mean their information is sent twice. 
+
+For example, the GOV.UK Notify team discovered that a number of users were receiving invitations twice, because the person sending them was double clicking the “send” button. 
+
+If you are working in production and research shows that users are frequently sending information twice, you can configure the button to ignore the second click. 
+
+To do this, include the `preventDoubleClick` attribute as shown in this example.
```

Comment:
I really don't want to overcomplicate this but there's two ways to use this feature.

  HTML data attribute:

   ```html
  <button class="govuk-button" data-prevent-double-click="true">
    Submit
  </button>
  ```

   Nunjucks macro:

   ```js
  {{ govukButton({
    text: "Submit",
    preventDoubleClick: true
  }) }}

Reply:
It may be that we only need to explain the macro and let the user figure out the rest if they dont use Nunjucks but thought it was worth mentioning, sorry

Reply:
OK I've tried to clarify this slightly using the wording discussed with @nickcolley and @hannalaakso